examples/httpserver/bin/hello_world_server.dart

```diff
@@ -4,22 +4,24 @@
 
 // Replies "Hello, world!" to all requests.
 // Use the URL localhost:4040 in your browser.
-
+// #docregion
 import 'dart:io';
 import 'dart:async';
 
 Future main() async {
-  var requestServer =
-      await HttpServer.bind(InternetAddress.LOOPBACK_IP_V4, 4040);
-  print('listening on localhost, port ${requestServer.port}');
+  // #docregion bind
+  var server = await HttpServer.bind(
+    InternetAddress.LOOPBACK_IP_V4,
+    4040,
+  );
+  // #enddocregion bind
+  print('Listening on localhost:${server.port}');
 
-  try {
-    await for (HttpRequest request in requestServer) {
-      request.response
-        ..write('Hello, world!')
-        ..close();
-    }
-  } catch (e) {
-    print(e.toString);
```

Comment:
The code excerpt in the tutorial page didn't contain a try-catch statement. I've reverted the code to match the excerpt, because

- Catching an exception and only printing a message is bad practice. (Note only that, but the code is wrong; it prints the value of the toString function).
- There are better examples of error handling elsewhere in the tutorial.

Reply:
Makes sense. We'll need to update the dart-tutorial-samples repo to match, of course. (But that's true everywhere.)

Reply:
#467

Reply:
Would you prefer that we point to the files in this repo? The files themselves will contain `#docregion` and `#docplaster` tags. As for a zip of the projects, it can be relatively easy to create a zip of the sources with those tags removed. We could then host those project .zip files from dartlang.org (along with the rest of the site files).

Reply:
I don't feel strongly about where we point to, as long as (1) those tags aren't visible and (2) we're careful about keeping everything in sync.